adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -898,6 +964,36 @@ end`
 		wellknown.CORS:                      corsFilter,
 	}
 
+	if endpointType == constants.AwsLambda {
+
+		var mode awslambdav3.Config_InvocationMode
+
+		if strings.ToUpper(conf.Envoy.AwsLambda.InvocationMode) == "SYNCHRONOUS" {
```

Comment:
As synchronous term is reused, lets make it a constant.

Reply:
Fixed by 9b38827